### DISCUSSION TITLE
How do you auto complete singatures not just function names, like in the demo video?

### DISCUSSION THREAD

**Discussion by Frau2015** [hrsh7th/nvim-cmp#544] [Q&A] [answered]
How do I get it to ghost text the entire signature not the just the function name?
<img width="623" alt="Screen Shot 2021-11-17 at 8 41 43 PM" src="https://user-images.githubusercontent.com/51972058/142353375-86f75075-e1e8-4ef3-987f-0488a0868203.png">
I want it like the demo:
<img width="810" alt="Screen Shot 2021-11-17 at 8 43 02 PM" src="https://user-images.githubusercontent.com/51972058/142353308-2bbcbf86-9323-48d8-a93f-1802d80c63f4.png">

**Comment by hrsh7th** [comment, marked as answer]
The ghost text is not a "function signature". This is a preview of the text after the completion is confirmed.

Anyway, It is depending on your language server. it is out of the scope of nvim-cmp's area.

**Comment by nima64** [comment]
Okay I found how to do it, you have to set callSnippet for lua configuration. I have it set to replace 

<img width="414" alt="Screen Shot 2021-11-23 at 10 09 53 AM" src="https://user-images.githubusercontent.com/53794276/143080748-d4805c36-eb6c-4f92-b170-028e1095fdf6.png">
t

**Reply by kofm** [reply]
Searched this for ages thanks!

**Reply by siduck** [reply]
@kofm  @nima64 , doesnt work for me! do i have to add something else to get this functionality?
